packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts

```diff
@@ -24,6 +24,7 @@ export const base = style([
       borderBottom: `6px solid ${tokens.kda.foundation.color.background['layer-3'].default}`,
       borderLeft: '6px solid transparent',
     },
+    zIndex: tokens.kda.foundation.zIndex.overlay,
```

Comment:
do you think we should add the zIndex tokens to atoms?

Reply:
I mean, yeah we could! Right now it has `[-1, 0, 1]` as options. If you think it's helpful, I will implement it in the atoms 👍

Reply:
I don't have a strong preference, so it's up to you 👍

Reply:
I think the values in the tokens are not generic and are specific to some components hence they should not be a utility class

Reply:
That's true, so leave them out of the atoms (at least for now)?